src/onboarding/Start.js

```diff
@@ -223,8 +223,8 @@ class StartContent extends React.PureComponent {
     if (!this.enoughBalance()) {
       return (
         <ActionInfo>
-          You need at least {weiToEther(MINIMUM_BALANCE)} ETH (you have{' '}
-          {Math.round(weiToEther((balance || 0).toFixed()) * 1000) / 1000} ETH).
+          You need at least {fromWei(MINIMUM_BALANCE)} ETH (you have{' '}
+          {Math.round(fromWei(balance || 0) * 1000) / 1000} ETH).
```

Comment:
This relies on an awkward implicit string to number conversion... which isn't nice.

Reply:
Maybe we could use `parseInt(x, 10)` to be safe?

Or we could add `fromWeiRounded(balance, digits = 2)` to web3-utils.js, that would handle the conversion and the rounding.

Reply:
`parseInt(x, 10)` is probably good enough for now; I think what we'd really want is a string rounding function to use in `fromWeiRounded()`: https://github.com/aragon/aragon/issues/307.